fields/types/select/SelectType.js

```diff
@@ -143,7 +143,19 @@ select.prototype.validateInput = function (data, callback) {
  */
 select.prototype.validateRequiredInput = function (item, data, callback) {
 	var value = this.getValueFromData(data);
-	var result = value !== undefined || (value === undefined && item.get(this.path)) ? true : false;
+	var result = false;
+	if (typeof value === 'string' && this.numeric) {
+		value = utils.number(value);
+	}
+	if (value === undefined) {
+		if (item.get(this.path)) {
+			result = true;
+		}
+	} else if (typeof value === 'string' || typeof value === 'number') {
```

Comment:
I'm guessing we're not looking in `this.map` because if the value made it this far, that has already been checked, and we just need to ensure a value is provided?

In which case, is there value in converting to a number at all? (ln. 148)